sar.c

```diff
@@ -170,7 +171,7 @@ void display_help(char *progname)
 	printf(_("\t-q\tQueue length and load average statistics\n"));
 	printf(_("\t-R\tMemory statistics\n"));
 	printf(_("\t-r [ ALL ]\n"
-		 "\tMemory utilization statistics\n"));
+		 "\t\tMemory utilization statistics\n"));
```

Comment:
Well observed! But this modification should not be part of this patch as it is not related to the addition of the "mountpoint" feature.